Python/crossfire/crossfire/occurrences.py

```diff
@@ -12,9 +12,13 @@ def __init__(self, client, id_state, id_cities=None, limit=None, format=None):
         self.next_page = 1
         self.yielded = 0
 
-        self.params = {"idState": id_state}
+        self.params = {"idState": id_state, "page": self.next_page}
         if id_cities:
-            self.params["idCities"] = id_cities
+            if isinstance(id_cities, list):
+                id_cities = {"idCities": [city for city in id_cities]}
+            else:
+                id_cities = {"idCities": id_cities}
```

Comment:
Não entendi o objetivo aqui. Basicamente esse código diz: se `id_cities` for uma lista, transforme ele numa lista idêntica. Ou seja… esse ceodigo não é o mesmo que:

```suggestion
            id_cities = {"idCities": id_cities}
```